ministryofjustice/laa-apply-for-criminal-legal-aid
# Present the offence class(es), comma separated

## Description of change
This is a tweak to adapt the way we show the offence class in the views, to the requirements of the design.

First, we unify the presentation of the class in just one single place (`OffencePresenter`). This presenter works with `Offence` instances thus always will have class attributes.

Additionally modify the existing `ChargePresenter` use the new `OffencePresenter` also to present information related to the offence, including the class.

Last, a new `OffenceCollectionDecorator` takes care of the offences dropdown, also presenting them with the new `OffencePresenter` so class is presented comma-separated as well.

## Screenshots of changes (if applicable)
<img width="591" alt="Screenshot 2022-09-22 at 17 09 48" src="https://user-images.githubusercontent.com/687910/191798555-d43eff8d-210e-43af-8c5e-649aa04b77aa.png">
<img width="755" alt="Screenshot 2022-09-22 at 17 12 57" src="https://user-images.githubusercontent.com/687910/191798653-f8a449aa-14de-412e-a442-cdfdfc0be968.png">
<img width="710" alt="Screenshot 2022-09-22 at 17 13 13" src="https://user-images.githubusercontent.com/687910/191798708-23d29550-a576-47a7-b144-be4e2b1f38ec.png">

## How to manually test the feature
See in the dropdown the different classes, and add some with one or more classes to see them in the basket too.
